src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -679,15 +687,13 @@ int MPIDI_OFI_init_local(int *tag_bits)
 
     MPIDI_OFI_global.num_vnis = num_vnis;
 
-    /* Create MPIDI_OFI_global.ctx[0] first  */
-    mpi_errno = create_vni_context(0);
-    MPIR_ERR_CHECK(mpi_errno);
-
-    /* Creating the additional vni contexts.
+    /* Creating the vni contexts.
      * This code maybe moved to a later stage */
-    for (int i = 1; i < MPIDI_OFI_global.num_vnis; i++) {
-        mpi_errno = create_vni_context(i);
-        MPIR_ERR_CHECK(mpi_errno);
+    for (int i = 0; i < MPIDI_OFI_global.num_vnis; i++) {
+        for (int j = 0; j < MPIDI_OFI_global.num_nics; j++) {
+            mpi_errno = create_vni_context(i, j);
```

Comment:
Maybe replace `i` and `j` with `nic` and `vni`? I get really confused which is which at this point 😄

Reply:
I agree. I did that elsewhere, but apparently didn't here.